### PR TITLE
Add official support for Django 4.1, 4.2 and Python 3.11

### PR DESCRIPTION
**Pull request by browniebroke**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -15,8 +15,8 @@ jobs:
       fail-fast: false
       max-parallel: 5
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10']
-        django-version: ['2.2', '3.0', '3.1', '3.2', '4.0', 'main']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        django-version: ['2.2', '3.0', '3.1', '3.2', '4.0', '4.1', '4.2', 'main']
```
Comment (WhyNotHugo) [inline .github/workflows/test.yml:19]: Can you remove `3.0`, `3.1` and `4.1` from here? Otherwise this matrix is getting a bit too bit.

**Review by WhyNotHugo — CHANGES_REQUESTED** [review]
Thanks, looks good. I'd just remove a few older versions from CI to avoid having too many jobs.

File: .github/workflows/test.yml (hunk 1 above)
Comment (browniebroke) [reply .github/workflows/test.yml:19]: I assume you mean `4.0` instead of `4.1` (which I've just added)?

Comment (WhyNotHugo) [reply .github/workflows/test.yml]: Yes, that, thanks!

**Pull request merged**